Mangosta/ViewControllers/Me.swift

```diff
@@ -20,12 +20,13 @@ class Me: UITableViewController, LoginControllerDelegate {
 		// TODO: when implementing vCard XEP-0054 add the FN field here
 		self.accountJID.text = self.xmppController?.xmppStream.myJID.bare()
 	}
-	@IBAction func signOut(sender: AnyObject) {
+	@IBAction func didPressSignOutButton(sender: AnyObject) {
```

Comment:
I dislike this change of name. `signOut` is a better fit for an `IBAction` method, because it describes an **action**, which is what Apple suggests to use in this case. On the other hand, `didPressSignOutButton` is more of a `delegate` method, which is not this case.